### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by NachoKiman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,39 @@
-# Democrafy web-client
+# Democrafy server
+This is one of the user applications of the Democrafy System. This is the web application that users can use to see
```
Comment (juliangalati) [inline README.md]: Check for common comments in the other projects.  
Also, I think we should clarify (in each readme) that **Democrify®** consists of three projects.

Comment (NachoKiman) [reply README.md]: Los comentarios comunes son intencionales y justamente sirven para aclarar como funciona el Democrafy System. Se compone de una aplicacion server y varias aplicaciones de usuario. Esta todo explicado en la seccion Democrafy System.

Comment (juliangalati) [reply README.md]: Now I see the confusion. What I meant was that if you fix something that is copied pasted, don't forget to fix it in the other projects.

Comment (NachoKiman) [reply README.md]: no hubo cambios

**Pull request merged**